Thu Apr 22 20:58:32 CEST 1999  Marc Espie  <espie@cvs.openbsd.org>
	* configure.in (openbsd): Factorize xmake_file.
	(ix86 openbsd): trim obsolete comment.
	(vax openbsd): typo.

--- gcc/configure.in.orig	Thu Apr 22 20:54:36 1999
+++ gcc/configure.in	Thu Apr 22 20:57:00 1999
@@ -518,6 +518,8 @@
 		tmake_file="t-libc-ok t-openbsd"
 		# avoid surprises, always provide an xm-openbsd file 
 		xm_file=${cpu_type}/xm-openbsd.h
+		# don't depend on processor x-fragments as well
+		xmake_file=none
  		if test x$enable_threads = xyes; then
  			thread_file='posix'
 			tmake_file="${tmake_file} t-openbsd-thread"
@@ -624,7 +626,6 @@
 	alpha*-*-openbsd*)
 		# default x-alpha is only appropriate for dec-osf.
 		target_cpu_default="MASK_GAS"
-		xmake_file=none
 		;;
 		
 	alpha*-dec-osf*)
@@ -1147,7 +1148,6 @@
 changequote(,)dnl
 	i[34567]86-*-openbsd*)
 changequote([,])dnl
-		# Remove when the math emulator is fixed
 		# we need collect2 until our bug is fixed...
 		use_collect2=yes
 		;;
@@ -2349,12 +2349,10 @@
 	mips*el-*-openbsd*)	# mips little endian
 		target_cpu_default="MASK_GAS|MASK_ABICALLS"
 		tm_file=mips/openbsd.h
-		xmake_file=none
 		;;
 	mips*-*-openbsd*)		# mips big endian
 		target_cpu_default="MASK_GAS|MASK_ABICALLS"
 		tm_file=mips/openbsd-be.h
-		xmake_file=none
 		;;
 	mips-sony-bsd* | mips-sony-newsos*)	# Sony NEWS 3600 or risc/news.
 		tm_file="mips/news4.h ${tm_file}"
@@ -2723,7 +2721,6 @@
 		;;
 	powerpc-*-openbsd*)
 		tmake_file="${tmake_file} rs6000/t-rs6000 rs6000/t-openbsd"
-		xmake_file=none
 		;;
 	powerpc-*-beos*)
 		cpu_type=rs6000
@@ -3239,7 +3236,7 @@
 		float_format=vax
 		;;
 	vax-*-openbsd*)
-		tmake_file="${tm_file} vax/t-openbsd"
+		tmake_file="${tmake_file} vax/t-openbsd"
 		;;
 	vax-*-ultrix*)			# vaxen running ultrix
 		tm_file="${tm_file} vax/ultrix.h"
